### ISSUE TITLE
Excuse me, are there any experimental results of training data2vec with and without additional data?

### ISSUE THREAD

**Issue by xiexukang** [HolgerBovbjerg/data2vec-KWS#1]

**Comment by HolgerBovbjerg** [comment]
Hi, 

Yes, I will add the results to the Readme 
Sorry for the late reply

**Comment by xiexukang** [comment]
when I train 200 epoch , the accuracy as follows :./runs/data2vec/last.pth with accuracy 93.41477576927412.
Saved ./runs/data2vec/last_encoder.pth with accuracy 93.41477576927412. Is the result similar to yours?

**Comment by HolgerBovbjerg** [comment]
It depends on which configuration you are using.
Is it with or without Data2Vec pretraining, and are you using all Speech Commands training data or only 20%?

**Comment by xiexukang** [comment]
I followed your readme to operate，the result is Data2Vec pretraining

**Comment by HolgerBovbjerg** [comment]
If you are using the KWT-1 model, pretraining on speech commands pretraining set, and finetuning on the reduced training set (20 %) then your result is similar to what I achieve. Try to do the baseline and compare the two, there should be a significant jump in accuracy.
